Stdin/README.md

```diff
@@ -0,0 +1,123 @@
+# Stdin Interface
+
+**Status**: Proposed
+
+| Name      | ID                                                                |
+| :-------- | :---------------------------------------------------------------- |
+| Stdin     | 8796536366ee21852db56dccb60bc564598b618c865fc50c8b1ab740bba128e3  |
+
+
+## Description
+
+Allows to get input from user for basic types like integers, addresses, public keys, strings, cells.
+
+## Functions
+
+`getStr` - prints prompt message to the user and returns string entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: bytes - string entered by user
+
+`getInt` - prints prompt message to the user and returns singed integer entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: int256 - integer entered by user
+
+`getUint` - prints prompt message to the user and returns unsinged integer entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: uint256 - integer entered by user
+
+`getAddr` - prints prompt message to the user and returns address entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: uint256 - signed integer entered by user
+
+`getCell` - prints prompt message to the user and returns TON address entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: cell - root cell of tree of cells
+
+## Declaration in Solidity
+
+```jsx
+interface IStdin {
+
+	function getStr(uint32 answerId, string prompt) external returns (string value);
```

Comment:
Suggest to use the following design pattern when the browser (or other interface implementor) must responds to the debot message:
1) Instead of uint32 answerId introduce two parameters `responseFunctionId` and `requestId` to specify response handler function and some response id that will glue response invocation with specific get request.
2) We need a declarations for response functions in this doc.

Reply:
Than we need two versions or interfaces - for c++ and solidity. c++ doesn't need anything except `answer_id` (function id is stored in saved awaiting frame).

Reply:
My view - we don't need `requestId` because we can define different `answerId` for different interface calls.

Reply:
Response functions looks different in c++ and solidity. In C++ you will use `co_await` to receive result synchronously. Thus there is no declaration for response in c++. For Solidity you can see an example of response function in Example.sol.